tmux-up

```diff
@@ -38,8 +38,12 @@ then
   exit 0
 fi
 
+CONF_DIR=${TMUXUP_CONF_DIR:-$HOME/.tmux}
```

Comment:
I wouldn't bother allowing users to set `$TMUXUP_CONF_DIR` to override the default. Pointing straight to `$HOME/.tmux/` seems fine.